pkg/dockerregistry/server/auth.go

```diff
@@ -161,12 +161,12 @@ type errAccessDenied struct {
 
 func (e *errAccessDenied) Error() string {
 	if e.err != nil {
-		return fmt.Sprintf("Origin client error: %v", e.err)
+		return fmt.Sprintf("origin client error: %v", e.err)
 	}
 	if len(e.reason) > 0 {
-		return fmt.Sprintf("Origin access denied: %s", e.reason)
+		return fmt.Sprintf("origin access denied: %s", e.reason)
 	}
-	return fmt.Sprintf("Origin access denied")
```

Comment:
> I'm not sure we want origin in these messages, actually

What about keeping `origin` just for the *client error* (there are other clients used in the registry code) and removing it from all the others?